src/event-replay/helpers.ts

```diff
@@ -42,36 +44,57 @@ export async function findTsvBlockHeight(filePath: string): Promise<number> {
  * @returns Genesis block data
  */
 export async function findBnsGenesisBlockData(filePath: string): Promise<BnsGenesisBlock> {
+  const genesisBlockMessage = await getGenesisBlockData(filePath);
+  const bnsGenesisBlock = getBnsRegistrationEvent(genesisBlockMessage);
+  return bnsGenesisBlock;
+}
+
+export async function getGenesisBlockData(filePath: string): Promise<CoreNodeBlockMessage> {
```

Comment:
A more explicit name would be best
```suggestion
export async function getGenesisBlockDataFromTsv(filePath: string): Promise<CoreNodeBlockMessage> {
```

Reply:
Yes, I believe this does cover both because on event replay, each new block event will run handleBnsImport in the /new_block handler. I'll right another test for this specifically